charts/tidb-cluster/templates/config/_drainer-config.tpl

```diff
@@ -11,7 +11,7 @@ detect-interval = {{ .Values.binlog.drainer.detectInterval | default 10 }}
 data-dir = "/data"
 
 # a comma separated list of PD endpoints
-pd-urls = "http://{{ template "cluster.name" . }}-pd:2379"
+pd-urls = "{{ if .Values.enableTLSCluster }}https{{ else }}http{{ end }}://{{ template "cluster.name" . }}-pd:2379"
```

Comment:
define scheme as a var like: https://github.com/pingcap/tidb-operator/pull/782/files#diff-d636dd7eac2fffa04bb4a9d148d705afR27

Reply:
It's an YAML template, not a shell script here

Reply:
You can define a `SCHEME` macro in `_helpers.tpl` to avoid repeating the `if-else` block again and again.